yurt/yurt_core/env.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
+
+import click
+import os
+import re
+import yaml
+from cookiecutter.main import cookiecutter
+from shutil import copy2, make_archive
+from yurt.yurt_core.utils import get_iter, find_file_in_ancestor
+from zipfile import ZipFile
+
+
+# Helpers
+
+def in_yurt_directory():
+    return os.path.exists('./docker-compose.yml') and \
+           os.path.exists('./django_app') and \
+           os.path.exists('./envs')
+
+
+def add():
+    # Create env var file in envs
+    os.chdir('./envs')
+    result_path = cookiecutter('gh:yeti/yurt_template-envvars')
+    environment = re.search(r'envs/(.+)\.env\.d$', result_path).group(1)
+
+    # Copy docker-compose.remote.yml
+    os.chdir('..')
+    target_file = './docker-compose.{}.yml'.format(environment)
+    copy2(
+        './docker-compose.remote.yml',
+        './docker-compose.{}.yml'.format(environment)
+    )
+
+    # Edit copy with new env vars
+    with open(target_file, 'r+') as target_file_obj:
+        compose_contents = yaml.load(target_file_obj.read())
+        target_file_obj.seek(0)
+        target_file_obj.truncate()
+        env_file = './envs/{}.env'.format(environment)
+
+        all_services = compose_contents.get('services', {})
+        for key, _ in get_iter(all_services):
+            all_services[key]['env_file'] = [env_file]
+        compose_contents['services'] = all_services
+        yaml.dump(compose_contents, target_file_obj, default_flow_style=False)
+
+
+def import_func(src):
+    click.echo('==> Extracting {} into ./envs'.format(src))
+    with ZipFile(src) as zip_stream:
+        zip_stream.extractall('./envs')
+    click.echo('==> Success!')
+
+
+def export(filename):
+    click.echo('==> Exporting .envs/ into ./{}.zip'.format(filename))
+    make_archive(filename, 'zip', './envs')
+    click.echo('==> Success!')
+    click.echo('==> Upload this zip archive to a vault. To import:')
+    click.echo('👉  yurt env import <path-to-archive>')
+
+
+class EnvCommand:
```

Comment:
Don't want to overdo the architecture but I'm curious if you should have an envs folder then have commands.py and helpers.py - Feels a little weird having this class in the middle of the file.